src/components/ui/Input.js

```diff
@@ -59,6 +91,7 @@ const StyledInput = styled(Input)`
       width: 100%;
       outline: none;
       height: 44px;
+      padding-left: ${(props) => (props.prepend ? '10px' : 0)};
```

Comment:
Do you have the equivalent right padding for append? How does append look?

Reply:
That line adds space between the gray bar and the "Search for..." text
![Screen Shot 2021-04-01 at 5 25 21 PM](https://user-images.githubusercontent.com/46976119/113360351-3f41c700-930f-11eb-8067-a3fb83b0cac3.png)

This is what it looks like with no padding for the append (how the code currently is)
![Screen Shot 2021-04-01 at 5 24 11 PM](https://user-images.githubusercontent.com/46976119/113360304-1a4d5400-930f-11eb-96c3-ac28e3da9916.png)
If I add the 10px of padding to the append, it looks like this (too much space between left outline and text)

![Screen Shot 2021-04-01 at 5 24 47 PM](https://user-images.githubusercontent.com/46976119/113360330-2a653380-930f-11eb-8ab6-7ef0a6fdd2c7.png)


Could be wrong tho